app.js

```diff
@@ -0,0 +1,34 @@
+//Require express
+const { response, request } = require("express");
+
+const express = require("express");
+
+const app = express();
+
+//Link public folder
+app.use(express.static("public"));
+
+//Creating routes:
+
+//Home
+app.get("/home", (request, response) => {  
+  response.sendFile(__dirname + "/views/home.html");   //__dirname = localhost:3000
```

Comment:
Actually `_dirname` stands for the path of the directory folder so in this case it will be something like `C:\Users\Eveline\Ironhack\labs\lab-express-basic-site`